design-patterns/domain-driven-design/pattern1/README.md

```diff
@@ -0,0 +1,194 @@
+
+If you are confused between Bounded Context, Entity, and the Aggregate. These terms correlate 1:1 for the purpose of this exercise. I will use the term Entity here to mean Root Entity which is an Aggregate in DDD. And, Bounded Context can have one or more than one Entity.
+
+
+
+
+## Create a Uni
+
+1. Login to [Vendia Share](https://share.vendia.net/login). 
+
+Don’t have a login, you can [sign up](https://share.vendia.net/) now. 
+
+2. Create Uni
+
+Click the `Create Universal Application` button on the top-right. Next, use the `Create your own` option. 
+
+3. Fill out details in the Uni creation wizard. 
+
+1. **Step 1:** Give your Uni a name. 
+Prefix your Uni name with prefix `test-` so its name can be reused. Pick a unique Uni name since all Starter and Individual Tier users will share the vendia.net namespace, which will cause naming collisions.
+Since this Uni will be owned and operated by the CRM Team. I will give it the name `test-CRM-<append-unique-suffix>`. I will simply refer to it as the `CRM`. Click `Next`. 
+
+2. **Step 2:** Fill out the following information to create a Uni node. 
+
+```
+Node Name: "CRM". // you can change the name.
+Node Description: "Node dedicated to the CRM Domain". // You can change the description.
+Cloud Server Provider: "AWS".
+Node Region:"us-east-1". //you can change it to other regions.
+Auth Option: "Vendia User". // Let's use Vendia User
+```
+
+
+3. **Step 3:** Provide the Uni schema. Copy and paste the Uni schema from the schema.json file in the schema folder. I have already designed the schema based on model we described above. But, this is the step in which you will spend most time while constructing Uni for your business problem.
+
+   Let’s review few important things in the schema that you will reuse in most of your Unis.
+
+Indexes on the `CustomerAccount`.
+
+```json
+"x-vendia-indexes": {
+       "CustomerAccountIdIndex": [
+           {
+               "type": "CustomerAccount",
+               "property": "customerId"
+           }
+       ]
+   }
+
+```
+The Singleton Value Object `Schema Owner`. You can store schema and Uni metadata in it.
+
+```json
+"SchemaOwner": {
+           "type": "object",
+           "description": "Schema Owner Details Singleton Value Object",
+           "properties": {
+               "group": {
+                   "type": "string"
+               },
+               "email": {
+                   "type": "string"
+               }
+           }
+       }
+
+```
+
+The `CustomerAccount` entity and its structure.
+
+```json
+"CustomerAccount": {
+           "description": "Customer Account",
+           "type": "array",
+           "items": {
+               "type": "object",
+               "properties": {
+		…
+```
```

Comment:
The coloring gets weird here

Reply:
it's because I am not copying the fill customer account. I added ellipsis to imply there is more. That is why.

Reply:
But, I checked the final preview. No issues with it.

Reply:
i removed the ellipsis